adding @fundamental-ngx/core through angular cli fails

#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
1. create a new app using `ng new myapp`
2. `cd myapp`
3. `ng add @fundamental-ngx/core`

getting the error:
```
ℹ Using package manager: npm
⠇ Searching for compatible package version...Invalid peer dependency @angular/common found in package.
Invalid peer dependency @angular/core found in package.
Invalid peer dependency @angular/forms found in package.
Invalid peer dependency @angular/animations found in package.
Invalid peer dependency @angular/router found in package.
Invalid peer dependency @angular/cdk found in package.
⚠ Unable to find compatible package. Using 'latest' tag.
⚠ Package has unmet peer dependencies. Adding the package may not succeed.

The package @fundamental-ngx/core@0.33.4 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
An unhandled exception occurred: Cannot find module '@angular/cdk/schematics'
```

My expectation would be either we document that we require `@angular/cdk` or we offer adding it while adding our package

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

Angular:
Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1302.5
@angular-devkit/build-angular   13.2.5
@angular-devkit/core            13.2.5
@angular-devkit/schematics      13.2.5
@angular/cli                    13.2.5
@schematics/angular             13.2.5
rxjs                            7.5.4
typescript                      4.5.5

--- a/libs/core/schematics/ng-add/index.ts
+++ b/libs/core/schematics/ng-add/index.ts
@@ -27,7 +27,6 @@ export function ngAdd(options: any): Rule {
 function addDependencies(options: Schema): Rule {
     return (tree: Tree, context: SchematicContext) => {
         const ngCoreVersionTag = getPackageVersionFromPackageJson(tree, '@angular/core');
-        const ngCdkVersionTag = getPackageVersionFromPackageJson(tree, '@angular/cdk');
         const dependencies: NodeDependency[] = [];
 
         if (!hasPackage(tree, '@angular/forms')) {
@@ -49,7 +48,7 @@ function addDependencies(options: Schema): Rule {
         if (!hasPackage(tree, '@angular/cdk')) {
             dependencies.push({
                 type: NodeDependencyType.Default,
-                version: `${ngCdkVersionTag}`,
+                version: `${ngCoreVersionTag}`,
                 name: '@angular/cdk'
             });
         }

